lib/meals.js

```diff
@@ -1,26 +1,111 @@
 module.exports = {totalCalories}
 
-
 var api_url = "https://obscure-harbor-85447.herokuapp.com/api/v1"
 
 $(document).ready(function(){
+  renderDiary()
+  renderFood()
+  removeFood()
+  addFoodToMeal()
+})
+
+function addFoodToMeal(){
+  $("#add-selected").on("click", function(event){
+    var foodId = event.target.getAttribute("foodId")
+    var mealId = event.target.getAttribute("mealid")
+    var checkedFoods = $("input:checked")
+    $.each(checkedFoods, function(index, food){
+      $.ajax({
+      type: "POST",
+      url: api_url + "/meals/" + mealId + "/foods/" + food.getAttribute("foodId")
+      })
+      .then(function(){
+        renderDiary()
+        renderFood()
+
+      })
+      .catch(function(error){
+        console.error(error)
+      })
+      console.log(food)
+    })
+  })
+}
+
+
+
+function renderFood(){
```

Comment:
if this is grabbing all the foods - should we call this `renderFoods` to be more explicit?

Reply:
good idea 👍